Header header
uint64 frame_seq
duration delta
detection_osnet/ProcessWindow[] windows